app/views/comments/load_comments.js.erb

```diff
@@ -1,4 +1,4 @@
-id = <%= @task.id %>
```

Comment:
What is your intention behind this change? Didn't we just ensure that the `@task` is properly authorised?

Reply:
That was an oversight by me. I undid that.

Reply:
Thanks for reverting that. I'm fine with that change, but in general would rather like to have the previous change be removed from Git (i.e., amending the commit that introduced the change). No need to do it here, but maybe for future occasions...